Code/Tools/AssetProcessor/native/unittests/AssetProcessorUnitTests.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+* Copyright (c) Contributors to the Open 3D Engine Project.
+* For complete copyright and license terms please see the LICENSE at the root of this distribution.
+*
+* SPDX-License-Identifier: Apache-2.0 OR MIT
+*
+*/
+
+#include <AzCore/UserSettings/UserSettingsComponent.h>
+#include <AzCore/Settings/SettingsRegistryMergeUtils.h>
+#include <native/unittests/AssetProcessorUnitTests.h>
+#include <native/unittests/UnitTestRunner.h> // for the assert absorber.
+#include <native/tests/MockAssetDatabaseRequestsHandler.h>
+
+namespace UnitTest
+{
+    AssetProcessorUnitTestAppManager::AssetProcessorUnitTestAppManager(int* argc, char*** argv)
+        : BatchApplicationManager(argc, argv)
+    {
+    }
+
+    bool AssetProcessorUnitTestAppManager::PrepareForTests()
+    {
+        if (!ApplicationManager::Activate())
+        {
+            return false;
+        }
+
+        // Not needed for this test, so disconnect
+        AssetProcessor::AssetBuilderInfoBus::Handler::BusDisconnect();

```

Comment:
Nit: this feels like it should be an option in the config. Its opinionated, easy to miss and has no control, so someone will be caught out in a future test expecting this behaviour.

Reply:
Will address in the next PR.